cdap-common/src/main/java/io/cdap/cdap/common/internal/remote/TaskWorkerHttpHandlerInternal.java

```diff
@@ -97,6 +98,8 @@ public class TaskWorkerHttpHandlerInternal extends AbstractHttpHandler {
   public TaskWorkerHttpHandlerInternal(CConfiguration cConf, DiscoveryService discoveryService,
       DiscoveryServiceClient discoveryServiceClient, Consumer<String> stopper,
       MetricsCollectionService metricsCollectionService) {
+    // set workdir location in cConf
+    cConf.set(TaskWorker.WORK_DIR, System.getProperty("user.dir"));
```

Comment:
where did we configure this `"user.dir"` property?

Reply:
modified to use cconf